### PR TITLE
Adds the Security Pouch

### PR DESCRIPTION
**Pull request by Flaborized**
## About the PR
Adds in a pouch that spawns in security officers', vice officers' and the HoS' pocket that contains all the items the security starter kit (that spawns in their bags) has. Replaces the aforementioned security starter kit. It works like nukeops pouches, and can be access from the pockets. Has six slots of storage for small items, such as handcuffs, flashes, grenades, donuts, stuff like that.

Importantly does **NOT** contain the changes to powercells my original patch had where the pouch was included. Powercells remain unchanged in this PR, you get exactly the same kit as the starter kit gives you. (Three handcuffs, one flash, one whistle, one 200PU powercell)

## Why's this needed? 
I think security is generally in need of cool kit that's not necessarily a "strong weapon" sorta thing. Having a unique little bag in your pocket that can hold a bit more stuff is cool, and would hopefully lead to players having more specific loadouts and methods of organizing all their sec gear. Introduces the choice of "What do I put in my pouch, what do I put in my belt, and what do I put in my bag?", letting players get a more unique "setup" based on their preferences.

[FEAT] [BALANCE] 

## Changelog

```
(u)Flaborized:
(+)Added the security pouch, a small pocket pouch containing all the kit previously contained in the Security Starter Kit. Starts in security officers', vice officers' and the HoS' pocket. Can be used to hold extra small items.
```

### CODE REVIEW

File: code/datums/jobs.dm [hunk 1]
```diff
@@ -216,23 +216,25 @@
 	slot_back = /obj/item/storage/backpack/withO2
 	slot_belt = /obj/item/gun/energy/taser_gun
 	slot_poc1 = /obj/item/device/pda2/hos
+	slot_poc2 = /obj/item/storage/security_pouch //replaces sec starter kit
 	slot_foot = /obj/item/clothing/shoes/swat
 	slot_head = /obj/item/clothing/head/helmet/HoS
 	slot_ears = /obj/item/device/radio/headset/command/hos
 	slot_eyes = /obj/item/clothing/glasses/sunglasses/sechud
-	items_in_backpack = list(/obj/item/storage/box/security_starter_kit) // Don't make them spawn with a full backpack again, please.
+	//items_in_backpack = list(/obj/item/storage/box/security_starter_kit) // Don't make them spawn with a full backpack again, please.
```
Comment (flrsh) [inline code/datums/jobs.dm:224]:
```suggestion
```

File: code/datums/jobs.dm [hunk 2]
```diff
@@ -216,23 +216,25 @@
...
 	slot_poc1 = /obj/item/device/pda2/hos
+	slot_poc2 = /obj/item/storage/security_pouch //replaces sec starter kit
 	slot_jump = /obj/item/clothing/under/rank/head_of_securityold
 	slot_suit = /obj/item/clothing/suit/armor/vest
 	slot_foot = /obj/item/clothing/shoes/swat
 	slot_head = /obj/item/clothing/head/helmet/HoS
 	slot_ears = /obj/item/device/radio/headset/command/hos
 	slot_eyes = /obj/item/clothing/glasses/sunglasses/sechud
-	items_in_backpack = list(/obj/item/storage/box/security_starter_kit) // Don't make them spawn with a full backpack again, please.
+	//items_in_backpack = list(/obj/item/storage/box/security_starter_kit) // Don't make them spawn with a full backpack again, please.
```
Comment (flrsh) [inline code/datums/jobs.dm:237]:
```suggestion
```

File: code/datums/jobs.dm [hunk 3]
```diff
@@ -423,7 +425,8 @@
 	slot_foot = /obj/item/clothing/shoes/swat
 	slot_ears =  /obj/item/device/radio/headset/security
 	slot_eyes = /obj/item/clothing/glasses/sunglasses/sechud
-	items_in_backpack = list(/obj/item/storage/box/security_starter_kit) // Don't make them spawn with a full backpack again, please.
+	slot_poc1 = /obj/item/storage/security_pouch //replaces sec starter kit
+	//items_in_backpack = list(/obj/item/storage/box/security_starter_kit) // Don't make them spawn with a full backpack again, please.
```
Comment (flrsh) [inline code/datums/jobs.dm:429]:
```suggestion
```

File: code/datums/jobs.dm [hunk 4]
```diff
@@ -1005,7 +1008,8 @@
 	slot_jump = /obj/item/clothing/under/misc/vice
 	slot_foot = /obj/item/clothing/shoes/brown
 	slot_ears =  /obj/item/device/radio/headset/security
-	items_in_backpack = list(/obj/item/storage/box/security_starter_kit) // Don't make them spawn with a full backpack again, please.
+	slot_poc1 = /obj/item/storage/security_pouch //replaces sec starter kit
+	//items_in_backpack = list(/obj/item/storage/box/security_starter_kit) // Don't make them spawn with a full backpack again, please.
```
Comment (flrsh) [inline code/datums/jobs.dm:1012]:
```suggestion
```

**Comment by Gannets** [comment]
I support anything that brings tacticool pouches to a wider audience

**Comment by ryebek** [comment]
Pouches cool, I support,

**Comment by Ley-bunned** [comment]
Solid support for sec's inventory woes, I dig it.

**Comment by Kyle2143** [comment]
Good stuff, made the changelog entry use the minor tag.

**Pull request merged**